src/position/position_utils.cairo

```diff
@@ -204,14 +330,23 @@ fn get_position_key(
     collateral_token: ContractAddress,
     is_long: bool,
 ) -> felt252 {
-    // TODO
-    0
+    let mut data = array![];
```

Comment:
this is an internal function, it should be implemented in internal impl

Reply:
Do we need to implement internals in libraries? Is there a problem in terms of security? I was just thinking that we need to implement internal functions for the contracts.

Reply:
I double checked and it's ok to let them as you did already, and yes we'll do internal only for contracts